src/components/Contact/contactlist.tsx

```diff
@@ -18,7 +27,7 @@ const ContactList = () => {
       ) : (
         <div className='mt-12 flex flex-col items-center gap-2'>
           <XCircleIcon className='h-16 w-16' />
-          <p>You don&apos;t have any contacts yet. Add some!</p>
+          <p>Sorry! No results found :(</p>
```

Comment:
We want to account for three separate states:

1. The user has no contacts
2. The user has a list of contacts
3. The user has contacts, but has filtered away all of them.

At the moment we're only covering 2 and 3.